docs/source/conf.py

```diff
@@ -213,10 +214,18 @@
     # Generate docs even if an item has no docstring.
     "undoc-members": True,
     # Don't document things with a leading underscore.
-    "private-members": False,
+    "private-members": True,
```

Comment:
I figure since the API docs are really only intended for engineers/developers, let's document a lot more than we typically do (i.e. private functions starting w/ an underscore, init constructors, and class inheritance)